tests/unit/adapters/listings-adapter-test.js

```diff
@@ -2,6 +2,7 @@ import { module, test } from 'qunit';
 import ListingsAdapter from 'shopify-buy/adapters/listings-adapter';
 import version from 'shopify-buy/version';
 import Promise from 'promise';
+import 'shopify-buy/isomorphic-btoa';
```

Comment:
I think that is the better way to go. I already have some wiring done in my node-test WIP, makes sense to add this to that since `btoa` might be needed in the future anywhere in the tests.

`src` will be fine since that's already handled in `shopify.js`